module/builder/collection/config.go

```diff
@@ -36,6 +42,8 @@ func DefaultConfig() Config {
 		ExpiryBuffer:            15,                              // 15 blocks for collections to be included
 		MaxPayerTransactionRate: 0,                               // no rate limiting
 		UnlimitedPayers:         make(map[flow.Address]struct{}), // no unlimited payers
+		MaxCollectionByteSize:   1000000,                         // 1MB
+		MaxCollectionTotalGas:   uint64(1000000),                 // 1M
```

Comment:
The limit has been defined [there](https://github.com/onflow/flow-go/pull/91/files#diff-4688e4f1e6bc5063d6bcb143912d69e213eff1019cf5caed6f75218349b70bc2R101), why defining again here?

Could we let me refer to the same number? 

I'm afraid one day we change one place and forgot to change the other.

Reply:
I just followed the convention in the code to have a DefaultConfig and command line support to change it if needed. I won't change it as it needs to change all other values as well. maybe something for the future to change, cc: @jordanschalm